azure-kusto-data/src/client.rs

```diff
@@ -112,8 +114,16 @@ impl KustoClient {
         ExecuteQueryBuilder::new(self.clone(), database.into(), query.into(), Context::new())
     }
 
-    pub(crate) fn prepare_request(&self, uri: &str, http_method: http::Method) -> Request {
-        let mut request = Request::new(uri.parse().unwrap(), http_method);
+    pub fn execute_command<DB, Q>(&self, database: DB, query: Q) -> ManagementQueryBuilder
```

Comment:
Open question - in most SDKs we also expose a general "execute" method, that sends to either query on command based on the "." prefix.
Do we want it here?
It's pretty useful, but it's still technically a heuristic.

Reply:
Though about that as well. The one downside i see here is that we might have to wrap the responses in an enum for the V1 and V2 response sets. Generally i am leaning more towards offering that API though.